mattermost-plugin/webapp/webpack.config.js

```diff
@@ -203,8 +203,18 @@ config.plugins.push(new webpack.DefinePlugin({
 }));
 
 if (NPM_TARGET === 'start:product') {
+    const url = new URL(process.env.MM_BOARDS_DEV_SERVER_URL ?? 'http://localhost:9006');
+
+    for (const rule of config.module.rules) {
+        if (rule.type === 'asset/resource' && rule.generator) {
+            rule.generator.publicPath = url.toString() + 'static/';
+        }
+    }
+
     config.devServer = {
-        port: 9006,
+        https: url.protocol === 'https',
```

Comment:
Small mistake: this should be `'https:'`. Took me some time to figure this out :)

Reply:
Good catch. I meant to ask you to test that since I'm not set up to test it locally.